dvc/objects/stage.py

```diff
@@ -52,11 +52,6 @@ def get_file_hash(path_info, fs, name, state=None):
         if hash_info:
             return hash_info
 
-    if not fs.exists(path_info):
-        raise FileNotFoundError(
-            errno.ENOENT, os.strerror(errno.ENOENT), path_info
-        )
-
```

Comment:
Some context: This was a temporary workaround for our old filesystems like `S3FileSystem` before we've migrated to `s3fs`. Now `fs` will just raise `FileNotFoundError`, so there is no need for this anymore.